Sources/SwiftTerm/Terminal.swift

```diff
@@ -3996,7 +3995,7 @@ open class Terminal {
             // Insert the line using the fastest method
             if bottomRow == buffer.lines.count - 1 {
                 if willBufferBeTrimmed {
-                    buffer.lines.recycle ().copyFrom (line: newLine)
+                    buffer.lines.recycle ()                    
                 } else {
```

Comment:
I am wondering - do you remember what this change was for?

Reply:
I'm not sure. Sorry. 

I remember some problems where I would get the wrong lines once the scrollback buffer couldn't fit the entire terminal history. But these problems could just be about the way I extracted lines.